packages/qr_models/lib/src/qr_models.dart

```diff
@@ -0,0 +1,42 @@
+// ignore_for_file: sort_constructors_first
+
+import 'package:intl/intl.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'qr_models.g.dart';
+
+/// {@template qr_models}
+/// Representation of the QR code that expires in a determinated time
+/// {@endtemplate}
+@JsonSerializable()
+class QRSeed {
+  /// Default constructor with the seed and the expiration time.
+  QRSeed(this.seed, this.expiresAt);
+
+  /// Represents the value for the QR code.
+  final String seed;
+
+  /// Represent the expiration time of the seed
+  final String expiresAt;
+
+  static const _dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
```

Comment:
suggestion: a static private constant feels wrong to me. I would either make it a seperate constant value or inline the format on line 25 since we're only using it one place.